ts/modules/item-series.src.ts

```diff
@@ -198,15 +198,17 @@ seriesType<Highcharts.ItemSeries>(
     // Prototype members
     {
         markerAttribs: void 0,
-        translate: function (this: Highcharts.ItemSeries): void {
+        translate: function (this: Highcharts.ItemSeries,
+            positions?: Array<number>
+        ): void {
             if (!this.slots) {
                 this.slots = [];
             }
             if (
                 isNumber(this.options.startAngle) &&
                 isNumber(this.options.endAngle)
             ) {
-                H.seriesTypes.pie.prototype.translate.call(this);
+                H.seriesTypes.pie.prototype.translate.call(this, positions);
```

Comment:
Did you consider using `apply(this, arguments)` instead? With `.call()` we may have similar issues in future if we decide to add more arguments.

Reply:
You're right! It seems reasonable.